### PR TITLE
New feedback page

### PR DESCRIPTION
**Pull request by egkrinkel**

### CODE REVIEW

**Comment by SallaDev** [comment]
> The branch `new-feedback-page` of https://github.com/SallaApp/theme-raed/pull/211 has Merge conflict❌

**Comment by SallaDev** [comment]
> The branch `new-feedback-page` of https://github.com/SallaApp/theme-raed/pull/211 has Merge conflict❌

**Comment by SallaDev** [comment]
> The branch `new-feedback-page` of https://github.com/SallaApp/theme-raed/pull/211 has Merge conflict❌

**Comment by SallaDev** [comment]
> The branch `new-feedback-page` of https://github.com/SallaApp/theme-raed/pull/211 has been synced from master..✅

File: src/views/pages/partials/single-comment.twig
Comment (jalmatari) [inline src/views/pages/partials/single-comment.twig:1]: remove the usage for: `hide_user`

**Review by jalmatari — CHANGES_REQUESTED** [review]

**Comment by SallaDev** [comment]
> The branch `new-feedback-page` of https://github.com/SallaApp/theme-raed/pull/211 has been synced from master..✅

**Comment by SallaDev** [comment]
> The branch `new-feedback-page` of https://github.com/SallaApp/theme-raed/pull/211 has been synced from master..✅

**Review by jalmatari — APPROVED** [review]

**Pull request merged**